contracts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chainlink/contracts",
-  "version": "0.3.1",
+  "version": "0.4.0-beta.0",
```

Comment:
Out of curiosity, will this get set to 0.4.0 prior to merge?

Reply:
Yep, good thought. I have it in the TODO in the original post as something that should be done prior to merge. We'll want to publish the beta package while leaving this PR open, test it, update the version to remove the beta tag, and finally publish the non-beta version.

Reply:
On second thought, I'm going to remove this since we'll be releasing to the beta channel/tag anyway first.